snappi_ixnetwork/resourcegroup.py

```diff
@@ -0,0 +1,202 @@
+import re
+
+class ResourceGroup(object):
+    """"""
+    _SPEED_MODE_MAP = {
+        'speed_1_gbps': 'normal',
+        'speed_10_gbps': 'tengig',
+        'speed_25_gbps': 'twentyfivegig',
+        'speed_40_gbps': 'fortygig',
+        'speed_50_gbps': 'fiftygig',
+        'speed_100_gbps':
+            '^(?!.*(twohundredgig|fourhundredgig)).*hundredgig.*$',
+        'speed_200_gbps': 'twohundredgig',
+        'speed_400_gbps': 'fourhundredgig'
+    }
+    
+    def __init__(self, ixnetworkapi):
+        self._api = ixnetworkapi
+        self._store_properties = []
+        self._layer1_conf = None
+        self.layer1_check = []
+        
+    def set_group(self):
+        self.layer1_check = []
+        self._layer1_conf = self._api.snappi_config.layer1
+        if self._layer1_conf  is None or len(
+                self._layer1_conf ) == 0:
+            return
+        
+        response = None
+        try:
+            payload = {"arg1": "/availableHardware", "arg2" : []}
+            url = '%s/availableHardware/operations/getChassisWithDetailedResouceGroupsInfo' \
```

Comment:
does this work for other than UHD as well? ex: for windows and linux API servers

Reply:
According to my understanding UHD is appliance base. This is is expose for chassis base as well. But current snappi_ixnetwork code is use this path only for UHD. I have a plan restructure those. This PR only for UHD.